src/app/shared/pipes/table-cell-shortener.pipe.ts

```diff
@@ -9,12 +11,10 @@ export class TableCellShortenerPipe implements PipeTransform {
     if (value == undefined) {
       return value;
     }
+    //Remove milliseconds if value is a timestamp with milliseconds
     if (value.match('\\b\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2}\\.\\d{3}\\b')) {
```

Comment:
It would be nice to declare the regex in a variable that describes what it is matching to.

Reply:
Its only used in one place, so this comment should provide enought context.

Reply:
But not less readable than: 
```js
    //Remove milliseconds if value is a timestamp with milliseconds
    if (value.match('\\b\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2}\\.\\d{3}\\b')) {
```

Reply:
Is it explained? yes, is it readable code? no. 
If the code is complex enough that is needs a comment it is probably complex enough to get a method.

e.g. 
```typescript
removeMilisecondsFromTimestamp(value: string): string
```

Reply:
You're right, i've fixed it